x-pack/plugins/spaces/public/management/edit_space/customize_space/customize_space_avatar.tsx

```diff
@@ -230,14 +341,32 @@ export class CustomizeSpaceAvatar extends Component<Props, State> {
 
     this.props.onChange({
       ...this.props.space,
+      customAvatarInitials: true,
       initials,
     });
   };
 
   public onColorChange = (color: string) => {
     this.props.onChange({
       ...this.props.space,
+      customAvatarColor: true,
       color,
     });
   };
 }
+
+const InitialsIcon: FunctionComponent = () => (
```

Comment:
Let's open an issue in EUI to have this added as a first-class icon that we can leverage in a followup

Reply:
Good idea, created https://github.com/elastic/eui/issues/4825